app/javascript/app/pages/login/forgot-password/forgot-password-component.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Icon from 'components/icon';
+import Input from 'components/input';
+import Button from 'components/button';
+import cwLogo from 'assets/cw-logo.svg';
+import { NavLink } from 'redux-first-router-link';
+import yellowButtonTheme from 'styles/themes/button/button-yellow.scss';
+import { LOGIN } from 'router';
+import styles from '../login-styles.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: null
+    };
+  }
+
+  render() {
+    const { dispatch, handleForgotPasswordClick } = this.props;
+    return (
+      <div className={styles.wrapper}>
+        <div className={styles.logoContainer}>
+          <Icon className={styles.logo} icon={cwLogo} />
+          <div className={styles.logoText}>NDC IMPLEMENTATION TRACKER</div>
+        </div>
+        <Input
+          label="Email"
+          inputType="text"
+          placeholder="Please add your email"
+          onChange={value => this.setState({ email: value })}
```

Comment:
Having an arrow function here means this component will be re-rendered every time as it evaluates as a new instance... so what about define a function in the class and pass it as a param? because you are also using it in two places (change and blur)